dbt/models/staging/amplitude/stg_amplitude__active_teacher_events.sql

```diff
@@ -0,0 +1,50 @@
+with amp_events as (
+    select * 
+    from {{ ref('base_amplitude__active_teacher_events') }}
+)
+, final as (
+    select
+        -- user ids and event time and type
+        amplitude_id,
+        ltrim(user_id::varchar, 0) as cdo_user_id,  -- Amplitude adds leading zeros to user_ids less than 5 chars long, strip them. (good find by natalia)
+        event_id, -- I don't know what this is, yet
+        event_time,
+        event_type,
+
+        case
+            when event_type = 'Teacher Viewing Student Work' then 'View Work'
```

Comment:
wrap in `lower()` to avoid CASE mismatch, such as...
```suggestion
            when lower(event_type) = 'teacher viewing student work' then 'View Work'
```